llvmlite/binding/executionengine.py

```diff
@@ -53,15 +55,25 @@ def __init__(self, ptr, module):
         module._owned = True
         ffi.ObjectRef.__init__(self, ptr)
 
-    def get_pointer_to_global(self, gv):
-        # XXX getPointerToGlobal is deprecated for MCJIT,
-        # getGlobalValueAddress should be used instead.
+    def get_pointer_to_function(self, gv):
+        warnings.warn(".get_pointer_to_function() is deprecated.  Use "
+                      ".get_function_address() instead.", DeprecationWarning)
         ptr = ffi.lib.LLVMPY_GetPointerToGlobal(self, gv)
         if ptr is None:
             raise ValueError("Cannot find given global value %r" % (gv.name))
         return ptr
 
-    get_pointer_to_function = get_pointer_to_global
```

Comment:
Thanks for the explanation. Could you update the docs? `get_pointer_to_global` is mentioned there.